### ISSUE TITLE
delword fails 4 testcases

### ISSUE THREAD

**Issue by rvjansen** [adesutherland/CREXX#221]
Look for DELWORD OK
failed in test 2
failed in test 12
failed in test 15
failed in test 16
DELWORD OK

**Comment by Peter-Jacob** [comment]
delword corrected: commit b364719391c7412a913b3c33bf4340d96d2b8584

test case 19 must be null-string as all chars are removed. In levelb it must be an exact match. In normal REXX a comparison of null-string '' with '1-blank ' is equal.  This is caused by: ```
the non-strict comparative operators will ignore leading or trailing blanks for string comparisons
```
Therefore I changed case 19:
`if delword(x,1) \= '' then say 'failed in test 19 '   `

once classic REXX is available we may re-visit this issue. 

Please verify and close the issue

**Comment by rvjansen** [comment]
verified and closed
